schemas/+lab/Path.m

```diff
@@ -4,6 +4,7 @@
 ---
 local_path          : varchar(255)               # local computer path
 net_location        : varchar(255)               # location on the network
+spock_path          : varchar(255)               # local spock path
```

Comment:
that should be bucket-path, no? Spock/Scotty are thee machines that access bucket

Reply:
ok, agreed.
Will change it right now